include/deal.II/grid/grid_tools.h

```diff
@@ -1239,6 +1239,34 @@ namespace GridTools
 
 
   /**
+   * Given an array of points use the global bounding box description obtained using
+   * GridTools::compute_mesh_predicate_bounding_boxto guess, for each of them,
+   * which process might own it.
+   *
+   * @param[in] global_bboxes : vector of bounding boxes describing the portion of
+   *  mesh with a property for each process.
+   * @param[in] points : the array of points to test.
+   *
+   * @param[out] tuple containing the following parts
+   *  - a vector indicized with ranks of processes. For each rank it contains
```

Comment:
Since this is a sentence punctuated by a ".", could you please capitalise the first letter of each item here and below.